test/e2e/series/Area.e2e.ts

```diff
@@ -11,7 +13,7 @@ describe('Area series', function() {
     var lines = element.all(by.css('.chart .area-series'));
     expect(lines.count()).toBe(2);
 
-    expect(lines.get(0)).toHaveExactClass('area-series mySeries0');
-    expect(lines.get(1)).toHaveExactClass('area-series mySeries1');
+    expect(lines.get(0).getAttribute('class')).toBe('area-series mySeries0');
```

Comment:
don't worry, i tried to use mocha and chai and had to do a similar approach like yours. however if you use jasmine in protractor the `toBe` methid can deal with promises out of the box ;)